kuma/core/apps.py

```diff
@@ -17,6 +17,5 @@ def language_mapping(self):
         """
         a static mapping of lower case language names and their native names
         """
-        from product_details import product_details
-        return dict([(lang.lower(), product_details.languages[lang]['native'])
+        return dict([(lang.lower(), settings.LOCALES[lang].native)
```

Comment:
Would you mind switching this to the dict literal syntax for both readability and [performance](https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/) improvements?